dask/dataframe/core.py

```diff
@@ -1432,23 +1432,39 @@ def to_dask_array(self, lengths=None, meta=None):
               on the first axis. These values are *not* validated for
               correctness, beyond ensuring that the number of items
               matches the number of partitions.
+        dtype : str or numpy.dtype, optional
+            The dtype to pass to :meth:`numpy.asarray` when called on
+            each partition.
+        na_value : Any, optional
+            The value to use for missing values. The default value depends
+            on `dtype` and the dtypes of the DataFrame columns.
+
+            Requires pandas >= 1.1.0.
         meta : object, optional
             An optional `meta` parameter can be passed for dask to override the
-            default metadata on the underlying dask array.
+            default metadata on the returned dask array.
+            Should not be used outside of advanced cases; setting `dtype`
+            is preferred.
 
         Returns
         -------
         """
+        if not PANDAS_GT_110 and na_value is not no_default:
+            raise NotImplementedError(
+                "na_value is not a valid argument for to_dask_array"
+                f"if pandas < 1.1.0. Pandas version is {pd.__version__}"
```

Comment:
Could you use the `PANDAS_VERSION` variable here instead of `pd.__version__` directly? 

https://github.com/dask/dask/blob/a1187b13321d69565b9c21359d739c239bd04c65/dask/dataframe/_compat.py#L8

It helps us when searching for various pandas-compatibility code paths